lib/sqlalchemy/dialects/postgresql/named_types.py

```diff
@@ -36,7 +38,7 @@ class NamedType(sqltypes.TypeEngine):
     DDLDropper: Type[NamedTypeDropper]
     create_type: bool
 
-    def create(self, bind, checkfirst=True, **kw):
+    def create(self, bind: _CreateDropBind, checkfirst: bool = True, **kw):
```

Comment:
please complete the typing of the function:
- kw should be typed as any
- return is None

this applies also to the other functions

Reply:
Alright, should be done :)

Reply:
thanks!